common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -468,6 +475,32 @@ def get_or_create_sjson(item, transcripts):
     return sjson_transcript
 
 
+def get_video_ids_info(edx_video_id, youtube_id_1_0, html5_sources):
+    """
+    Returns list internal or external video ids.
+
+    Arguments:
+        edx_video_id (str):
+        youtube_id_1_0 (str):
+        html5_sources (list):
+
+    Returns:
+        tuple: external or internal, video ids list
+    """
+    clean = lambda item: item.strip() if isinstance(item, basestring) else item
```

Comment:
what if item is not string?

Reply:
In that case item will be returned as it is.